vagrant/install.sh

```diff
@@ -4,11 +4,10 @@
 true
 SCRIPT_NAME="Install NcVM with Vagrant"
 # shellcheck source=lib.sh
-source /var/scripts/fetch_lib.sh || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
 
-check_command git clone https://github.com/nextcloud/vm.git
+install_if_not wget
+check_command wget https://raw.githubusercontent.com/nextcloud/vm/master/nextcloud_install_production.sh
 
-cd vm || exit && print_text_in_color "$IRed" "Could not cd into the 'vm' folder."
-
-yes no | sudo bash nextcloud_install_production.sh
```

Comment:
The vm folder needs to be local. Hence the code for that.

Reply:
I doubt that. We have nothing implemented to use the local scripts.
I am pretty sure that it just runs the script like everywhere else. (Loads the scripts during nextcloud_iinstall_production.sh directly from github)

Reply:
Well, have you actually tested it? 

I know for a fact the the current implementation works, so of you don't want to test the new one, please revert. :)

Reply:
Okay, I'll do 👍

Reply:
done